### PR TITLE
Steven edit profile

### PR DESCRIPTION
**Pull request by stevenbursztyn**

### CODE REVIEW

**Comment by katiejiang** [comment]
For edit profile, could you actually go ahead and add a field for editing GPA and FAFSA status?

**Comment by stevenbursztyn** [comment]
Sure, will add those changes in for GPA and FAFSA status. 

Also going ahead to add the major changes.

File: app/student/views.py [hunk 1]
```diff
@@ -85,3 +86,91 @@ def add_essay():
...
+    if str == 'False':
+        return False
+
+def bool_to_string(bool):
+    if bool:
+        return 'True'
+    else:
+        return 'False'
+
+@student.route('/profile/edit_profile', methods=['GET', 'POST'])
```
Comment (katiejiang) [inline app/student/views.py]: I like the route `/profile/edit` better.

**Comment by katiejiang** [comment]
LGTM

**Pull request merged**